paths/paths.go

```diff
@@ -87,7 +89,7 @@ func (paths *Path) InitPaths(cfg *Path) error {
 	// make sure the data path exists
 	err = os.MkdirAll(paths.Data, 0750)
 	if err != nil {
-		return fmt.Errorf("Failed to create data path %s: %v", paths.Data, err)
+		return errors.Wrapf(err, "Failed to create data path %s", paths.Data)
```

Comment:
Question: Why not use `%w` instead of `github.com/pkg/errors`?

Reply:
`github.com/pkg/errors` is archived considers itself replaced by the standard library functionality. https://github.com/pkg/errors is archived for example.

Reply:
Interesting, I missed that. `errors.Wrap` is quite common in the codebase, I'll migrate the rest of the code to `fmt.Errorf` then.

Reply:
I replaced all 32 occurrences in the entire repo (97bd94a3098995057bdd1ec6d7b250597152703c) and `github.com/pkg/errors` is no longer a direct dependency.